### ISSUE TITLE
Console crash when pasting parameter

### ISSUE THREAD

**Issue by ili101** [PowerShell/vscode-powershell#1675]
Issue Type: <b>Bug</b>

1. Create a new Untitled-1 Doc (PowerShell)
2. paste this code
``` PowerShell
function Test-Params
{
    Param(
        $Param1,
        $Param2
    )
}
Test-Params -Param1
```
3. Copy "Param2"
4. paste it over "Param1" (in the line `Test-Params -Param1`)
(if it's not crashing the first time repeat with "Param2")
5. crash...

![error](https://user-images.githubusercontent.com/6960531/50497418-d9837800-0a3f-11e9-9157-827d137b1d45.gif)

Extension version: 1.10.2
VS Code version: Code 1.30.1 (dea8705087adb1b5e5ae1d9123278e178656186a, 2018-12-18T18:12:07.165Z)
OS version: Windows_NT x64 10.0.17134

[EditorServices.log](https://github.com/PowerShell/vscode-powershell/files/2713579/EditorServices.log)

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 1800X Eight-Core Processor          (16 x 3593)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.95GB (8.26GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details>

**Comment by autosvet** [comment]
I can confirm the same behaviour with PS Core 6.1.0 and 6.1.1.

PS Extension version: 1.10.2
VSCode 1.30.1

**Comment by rkeithhill** [comment]
Thanks for submitting this.  I have a fix I'll be submitting shortly.  It might be a few weeks until there is a release though. In the meantime, if you don't use an untitled file then you shouldn't run into this bug.

**Comment by autosvet** [comment]
Thanks Keith.,
The workaround works for me. 
Just save the file, even as `Untitled-1.ps1` , and the console stops crashing after paste operation.

**Comment by rkeithhill** [comment]
Yeah, it is not the name per-se, it is that it is an "in-memory" document with no file backing it.

**Comment by rkeithhill** [comment]
I believe this bug to be fixed by PR merged in above.  The fix should be available in 1.10.3.  Should we wait for confirmation of the fix before closing this?

**Comment by rkeithhill** [comment]
I'm closing this since we believe it to be fixed. @ili101 you can re-open it if you still see this bug with 1.11.0.
